test/interpreter_functional/test_suites/run_pipeline/basic.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+import { expectExpressionProvider } from './helpers';
+import { delay } from 'bluebird';
+
+export default function ({ getService, updateBaselines }) {
+
+  let expectExpression;
+  describe('basic visualize loader pipeline expression tests', () => {
+    before(() => {
+      expectExpression = expectExpressionProvider({ getService, updateBaselines });
+    });
+
+    after(async () => {
+      await delay(50000);
```

Comment:
I think we should try to make this would work without sleeps.  If we do need sleeps though, most functional tests use `PageObjects.common.sleep(x)`.  Could use that to stay consistent.

Reply:
this one is definetely not needed, was just there for me so i see what happens (else everything was way to fast :D)

removing